gradle.properties

```diff
@@ -39,10 +39,12 @@ commonsLang3Version=3.12.0
 commonsMath3Version=3.6.1
 javaPoetVersion=1.12.0
 log4jVersion=2.17.2
-guavaVersion=30.1-jre
+guavaVersion=31.0.1-jre
 hamcrestVersion=2.2
 # AutoService kept on 1.0-rc6 to avoid annotation being retained in CLASS. See: https://github.com/FoundationDB/fdb-record-layer/issues/1281
 autoServiceVersion=1.0-rc6
+# AutoService for development
```

Comment:
Might be worth noting in a comment here that this is bumped during development to enable incremental builds.